Invoice_to_Payment_Process/Invoice_to_Payment_Process.md

```diff
@@ -109,6 +109,51 @@ graph LR
 
 ### 3.1 Message Level Response
 
+#### 3.1.1 Scope
+
+The scope of the Message Level Response is:
+
+1. The Message Level Response uses BIS Message Level Response 3.0 (see this link for the [Peppol business documentation|https://docs.peppol.eu/poacc/upgrade-3/profiles/36-mlr/]. The technical UBL message used is ApplicationResponse. See paragraph 5 for the message definition.
+2. The Message Level Response is sent by the customer AP (C3) on behalf of the customer (C4) to the supplier (C1) but can be handled by the supplier AP (C2) conform standard Peppol exchange mechanisms (including use of SML and SMP) and always refers to the envelop of the business document sent by the supplier.
+3. The SMP record for the MLR should use the sending AP as endpoint or route the message to the supplier AP (C2).
+4. The following errors are within the scope for a negative/rejecting Message Level Response:
+ 	* XML schema validation error
+	* Standard Compliance violations (e.g. empty elements not being allowed by UBL 2.1)
+	* Validation error of type fatal error
+	* Validation error of type warning. Warnings alone must NOT cause rejection of the business document (but they may be reported in addition to fatal errors)
+	* Wrong version of business document (Will be handled like validation error of type fatal error)
```

Comment:
What does this mean? Send NLCIUS instead of BIS3? Or send UBL2.3 instead of 2.1?

Reply:
List from https://docs.peppol.eu/poacc/upgrade-3/profiles/36-mlr/#_the_following_errors_are_within_the_scope_for_a_negativerejecting_message_level_response
I think this ia about mismatch of document registered and send. MSH at transport level SHOULD have thrown an error